content/pt/_collections/testimonials/index.md

```diff
@@ -1,34 +1,34 @@
 ---
 testimonials:
   -
-    testimonial: 'Nuxt offers a compelling solution and a great ecosystem to help you ship fullstack Vue apps that are performant and SEO friendly. The flexibility to choose between SSR and SSG is icing on the cake.'
+    testimonial: 'A Nuxt oferece uma solução convincente e um excelente ecossistema para ajudar você a entregar aplicações Vue completas que são funcionais e amigáveis ao SEO. A flexibilidade para escolher entre a Renderização no Lado do Servidor (SSR, sigla em Inglês) e Geração de Sítios Estáticos (SSG, sigla em Inglês) é a cereja sobre o bolo.'
```

Comment:
Testimonials should not be translated. Those sentences are quotes of their author and must not be translated or altered in any ways.